### PR TITLE
added admin route to go into edit mode

### PR DESCRIPTION
**Pull request by logan-anderson**
- `/admin/` route triggers Edit Mode, it's the most common pattern when it comes to log into a CMS 
- The environment variable `NEXT_PUBLIC_SHOW_EDIT_BTN` is set to 1 by default, set it to 0 if you want to hide Edit Mode button

## Demo
https://www.loom.com/share/3bd7bd809ac84954bf2f573fe3ca75fe

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -30,31 +31,35 @@ const EditToggle = (isInEditMode) => {
...
-          padding: 0.5rem 0.75rem;
-          font-weight: bold;
-          text-decoration: none;
-          display: inline-block;
-          border-bottom-left-radius: 0.5rem;
-          cursor: pointer;
-          font-size: 20px;
-        }
-      `}</style>
+      {(Number(process.env.NEXT_PUBLIC_SHOW_EDIT_BTN) || edit) && (
```
Comment (DirtyF) [inline pages/_app.js:34]: I understand `NEXT_PUBLIC_SHOW_EDIT_BTN` is a boolean, usage could be documented in the README

File: pages/admin.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import { Wrapper } from "../components/helper-components";
+import { useEditState } from "../utils/editState";
+
+const GoToEditPage: React.FC = () => {
+  const { setEdit } = useEditState();
+  const router = useRouter();
+  useEffect(() => {
+    setEdit(true);
+    router.back();
```
Comment (DirtyF) [inline pages/admin.tsx:12]: What happens if I'm on the homepage and I hit edit with this pattern?

**Review by DirtyF — APPROVED** [review]
Great addition, minor comments related to the pattern documentation.

File: pages/admin.tsx (hunk 2 above)
Comment (logan-anderson) [reply pages/admin.tsx:12]: Both the edit button and GoToEditPage do the same thing. They are just calling `setEdit(true)` So you would just go into edit mode as expected.

File: pages/_app.js (hunk 1 above)
Comment (logan-anderson) [reply pages/_app.js:34]: Pushed an update to the readme.

**Review by jbevis — APPROVED** [review]
Looks good Logan, nice work.

**Review by Enigmatical — APPROVED** [review]

**Pull request merged**